tests/Api/Admin/ProductVariantsTest.php

```diff
@@ -481,41 +481,6 @@ public function it_does_not_allow_to_update_product_variant_with_invalid_locale_
         );
     }
 
-    /** @test */
```

Comment:
seems like we are adding default locale automatically:

https://github.com/Sylius/Sylius/commit/33f7a1e11dbe53b2644fec9066c8f9a1e94439e1